content/ar/docs/setup/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: دليل الاستخدام
+main_menu: true
+weight: 20
+content_type: concept
+no_list: true
+card:
+  name: setup
+  weight: 20
+  anchors:
+  - anchor: "#بيئة-التعلم"
+    title: بيئة التعلم
+  - anchor: "#بيئة-الإنتاج"
+    title: بيئة الإنتاج  
+---
+
+<!-- General overview-->
```

Comment:
in the english version it was just overview
https://github.com/kubernetes/website/blob/main/content/en/docs/setup/_index.md?plain=1
TBH not sure if this comments are interpreted by the template engine to format the page so let's keep it as it was in the english version